java/src/com/google/idea/blaze/java/qsync/GeneratedSourcesStripper.java

```diff
@@ -139,8 +141,16 @@ private void removeSourcesWithMatchingAnnotations(CachedArtifact src, Path dest)
   @VisibleForTesting
   String getGeneratedValue(String fileName, ByteSource content) throws IOException {
     return ApplicationManager.getApplication()
-        .runReadAction(
-            (ThrowableComputable<String, IOException>) () -> getInReadAction(fileName, content));
+            .runReadAction(
+                    (ThrowableComputable<String, IOException>) () ->
```

Comment:
`IOException` is no longer required here

Reply:
Done, can you PTAL again?